bot/utils/messages.py

```diff
@@ -17,3 +24,66 @@ def replace_e(match: re.Match) -> str:
         return re.sub(r"(clyd)(e)", replace_e, username, flags=re.I)
     else:
         return username  # Empty string or None
+
+
+async def get_discord_message(ctx: Context, text: str) -> Union[Message, str]:
+    """
+    Attempts to convert a given `text` to a discord Message object and return it.
+
+    Conversion will succeed if given a discord Message ID or link.
+    Returns `text` if the conversion fails.
+    """
+    try:
+        text = await MessageConverter().convert(ctx, text)
+    except commands.BadArgument:
+        log.debug(f"Input '{text:.20}...' is not a valid Discord Message")
+
+    return text
```

Comment:
```suggestion
    try:
        return await MessageConverter().convert(ctx, text)
    except commands.BadArgument:
        return text
```

The logging here seems not necessary and is a bit spammy considering it will go off on every non-link invocation of `.uwu`, it's perfectly normal for the exception to occur and to be caught for messages without a valid message link.